internal/core/data/init.go

```diff
@@ -62,6 +62,19 @@ func (b *Bootstrap) BootstrapHandler(ctx context.Context, wg *sync.WaitGroup, st
 	mdc := metadata.NewDeviceClient(local.New(configuration.Clients["Metadata"].Url() + clients.ApiDeviceRoute))
 	msc := metadata.NewDeviceServiceClient(local.New(configuration.Clients["Metadata"].Url() + clients.ApiDeviceRoute))
 
+	// For Redis Streams MessageBus, we reuse the Redis instance running for the DB, which may have a password,
+	// so we need to get and use the DB credentials for the MessageBus connection.
+	if configuration.MessageQueue.Type == "redisstreams" {
```

Comment:
Do case insensitive for this?

Reply:
Nope, not consistent with what MessageBus factory is doing.